lib/lambda_ethereum_consensus/p2p/subscriber.ex

```diff
@@ -0,0 +1,33 @@
+defmodule LambdaEthereumConsensus.Subscriber do
```

Comment:
I think in Elixir we tend to follow the folder structure when naming modules. So this one would be `LambdaEthereumConsensus.P2P.Subscriber`